src/actions/NewSensorActions.js

```diff
@@ -10,4 +14,25 @@ const updateConfig = (configType, configField, value) => ({
   payload: { configField, configType, value },
 });
 
-export const NewSensorActions = { select, updateConfig };
+const updateCode = code => ({ type: NEW_SENSOR_ACTIONS.UPDATE_CODE, payload: { code } });
+
+const updateName = name => ({ type: NEW_SENSOR_ACTIONS.UPDATE_NAME, payload: { name } });
+
+const updateLogInterval = logInterval => ({
+  type: NEW_SENSOR_ACTIONS.UPDATE_LOG_INTERVAL,
+  payload: { logInterval },
+});
+
+const updateLoggingDelay = loggingDelay => ({
+  type: NEW_SENSOR_ACTIONS.UPDATE_LOGGING_DELAY,
+  payload: { loggingDelay },
+});
```

Comment:
could have these as a single action with a field in the payload for which field in the new sensor to update. I kind of like the explicitness .. but then the action to update the configs is consistent with that.

I dunno